pkg/features/kube_features.go

```diff
@@ -727,6 +727,15 @@ const (
 	// Subdivide the NodePort range for dynamic and static port allocation.
 	ServiceNodePortStaticSubrange featuregate.Feature = "ServiceNodePortStaticSubrange"
 
+	// owner: @gjkim42 @SergeyKanzhelev @matthyx @tzneal
+	// kep: http://kep.k8s.io/753
+	// alpha: v1.28
+	//
+	// Introduces the Sidecar containers, a new type of containers that start
```

Comment:
`Introduces sidecar containers, a new type of init container that starts before other containers but remain running during the lifecycle of the pod and will not block pod termination.`